source/assets/stylesheets/styles/_project_team.scss

```diff
@@ -27,13 +27,23 @@
   }
 }
 
-.sh24-team__headshot-container {
+.project-team__headshot-container {
   display: block;
   float: left;
   width: 20%;
```

Comment:
These widths should be extracted out to their own classes: we have `.project-team__headshot-container` as our base class, but at the moment it is 5 people by default. We will be overwriting this each time because we know that team sizes will vary for each project.